app/views/comments/_comment_actions.html.erb

```diff
@@ -6,6 +6,11 @@
   <% end %>
   <% unless comment.unreviewed? %>
     <li><%= link_to ts("Thread"), comment %></li>
+    <% if comment.depth > 0 %>
+    <li>
```

Comment:
Not blocking, but could you tweak the indenting on this slightly to make it clearer what the `if` encapsulates? 
```
<% if comment.depth > 0 %>
  <li>
    <%= link_to
  </li>
<% end %>
```